### PR TITLE
Update Glowstone version, update code to match new Glowstone API, fixes #769

### LINKED ISSUE #769: Error when a client pings the server [Glowstone]
Hello :)

I'm using glowstone (newest version, latest build by website) and only ProtocolSupport (newest jenkins version) in addition.
When the server starts and a client pings the server, the server won't answer (therefore no one can connect).

Everytime a connection should be established or a client pings, the following error will be thrown:

``
[/127.0.0.1:10100] kicked: read error: net.glowstone.net.pipeline.MessageHandler.getSession()Ljava/util/concurrent/atomic/AtomicReference;
``

I have tried the porkaxe method, and it didn't help.

### PR DESCRIPTION
**Pull request by MrPowerGamerBR**
According to @DoppelnullYT, the fix seems to be working

### CODE REVIEW

File: src/protocolsupport/zplatform/impl/glowstone/GlowStoneMiscUtils.java [hunk 1]
```diff
@@ -10,6 +10,8 @@
 import javax.crypto.Cipher;
 import javax.crypto.SecretKey;
 
+import com.destroystokyo.paper.profile.ProfileProperty;
```
Comment (7kasper) [inline src/protocolsupport/zplatform/impl/glowstone/GlowStoneMiscUtils.java]: ???

Comment (7kasper) [reply src/protocolsupport/zplatform/impl/glowstone/GlowStoneMiscUtils.java]: Ohh glowstone uses the paper api

Comment (MrPowerGamerBR) [reply src/protocolsupport/zplatform/impl/glowstone/GlowStoneMiscUtils.java]: @7kasper Glowstone uses Paper's PlayerProfile API

https://github.com/GlowstoneMC/Glowstone/commit/0cfbb0692a4aafb282a6d68b26f780102bdec666#diff-bd59b1c2be78b091d2e887be7f414fe2

Comment (MrPowerGamerBR) [reply src/protocolsupport/zplatform/impl/glowstone/GlowStoneMiscUtils.java]: (and you figured that out yourself before I commented that but GitHub didn't update the comments, whoops)

**Comment by 7kasper** [comment]
I approve this message.

**Pull request merged**